Add plugin: AutoMover

al0cam opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin:
https://github.com/al0cam/AutoMover

## Release Checklist
- [ ] I have tested the plugin on
  - [ ]  Windows
  - [ ]  macOS
  - [x]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
[containerEl.createEl("h1", { text: "AutoMover settings" });](https://github.com/al0cam/AutoMover/blob/f561257ac7d489b884eade197387432a3d854fc4/Settings/SettingsTab.ts#L17-L20)
[Don't add a top-level heading in the settings tab, such as "General", "Settings", or the name of your plugin.](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Only+use+headings+under+settings+if+you+have+more+than+one+section.)

[containerEl.createEl("h2", { text: "Automatic moving" });](https://github.com/al0cam/AutoMover/blob/f561257ac7d489b884eade197387432a3d854fc4/Settings/SettingsTab.ts#L22), [header.createEl("h2", { text: "Tutorial" });](https://github.com/al0cam/AutoMover/blob/f561257ac7d489b884eade197387432a3d854fc4/Settings/SettingsTab.ts#L70), [movingRulesContainer.createEl("h2", { text: "Regex moving rules", cls: "rule_header" });](https://github.com/al0cam/AutoMover/blob/f561257ac7d489b884eade197387432a3d854fc4/Settings/SettingsTab.ts#L131)
For section headings in settings use:
```ts
new Setting(containerEl).setName('name here').setHeading();
```

[text: "Search Criteria (String or Regex)",](https://github.com/al0cam/AutoMover/blob/f561257ac7d489b884eade197387432a3d854fc4/Settings/SettingsTab.ts#L136)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)

al0cam commented:
@joethei Thank you for the review.
I added the changes you proposed and merged them into the plugin.
After that I updated the community-plugins.json but i see now that it's not rebasable due to conflicts (my formatter deleted spaces where it found them)
Is that a problem or is is it ok as is?

joethei commented:
Just ignore the rebase error, I will do a rebase when merging this PR.

al0cam commented:
Ok, thank you.
